Sources/Apollo/AutomaticPersistedQueryInterceptor.swift

```diff
@@ -48,6 +48,8 @@ public struct AutomaticPersistedQueryInterceptor: ApolloInterceptor {
                                request: request,
                                response: response,
                                completion: completion)
+
+        chain.terminate()
```

Comment:
If we proceed with this solution, would it be reasonable to just terminate the chain in `handleErrorAsync()`? Or are there situations in which we want to handle and error without termination?

Reply:
`handleErrorAsync` doesn't mean the request chain has ended unfortunately; that's in the interceptor implementation and there is nothing in the request chain logic that ends a request with a call to `handleErrorAsync`. It's simply a way for you to get additional information about an error that has been encountered.